raiden/tests/utils/typing.py

```diff
@@ -0,0 +1,4 @@
+from raiden.app import App
+from raiden.utils.typing import Iterable, Tuple
+
+AppChannels = Iterable[Tuple[App, App]]
```

Comment:
What are the first `App` and the second `App`?  Are they this side and that side of the channel, or the ordering has no meaning?